src/services/postService.js

```diff
@@ -1,12 +1,61 @@
+const uuid = require("uuid");
 const { throwIfError } = require('../utilities/dynamoUtilities');
-const { sendPost } = require("../repository/postDAO");
+const postDAO = require("../repository/postDAO");
 
-async function createPost(username, text, score, title){
-    const post = await sendPost(username, text, score, title);
-    throwIfError(post);
-    return {message: "Post created successfully"};
+
+async function createPost(username, description, score, title){
+    const post = {
+        class: "post",
```

Comment:
I would define this as a constant somewhere; in the latest version of `dyanmoUtilities` there's a `CLASS_USER` & `CLASS_POST` that you can use.

Reply:
Good catch! Will update to use that once I start merging